group_project_v2/templates/html/activity/dashboard_detail_view.html

```diff
@@ -42,7 +42,7 @@
         {% endfor %}
       </tr>
         {% for user in group.users %}
-          <tr class="user-data-row data" data-group-id="{{group.id}}" data-fullname="{{user.full_name}}" data-email="{{user.email}}">
+          <tr class="user-data-row data{% if user.is_filtered_out %} filtered-out{%endif%}" data-group-id="{{group.id}}" data-fullname="{{user.full_name}}" data-email="{{user.email}}">
```

Comment:
@jbzdak I believe you should put css rule in GPv2 css, as it is part of GPv2 functionality. The rule of thumb is basically to put something **really** Apros specific into Apros css overrides, and try to keep all the CSS here.